Italian translation

Stefano1975t opened the pull request:
Added the translation for the italian language (it.json) and updated system.json with the new language.

pwatson100 commented:
Hi, Thank you you for the it.json.  One just a couple of things.  I need to merge this with my Release-Testing branch, which is where I  do all the development work before I commit to master and release.  I have made some additions to the language files in that branch so you might want to take a look.  Also slightly restructured the format of the languagefiles to alow me to use a new tool to manage them.  There is a conflict with the system.json as release-testing is version 2.1.1 but that is easy enough to sort out.  My main concern is you have made changes to my main.yml which I do not want processed.  
I am taking a copy of the it.json and adding it to release testing so if you check that branch you should see the changes. 
Thank you again for providing this translation.  
Regards,
  Paul.

Stefano1975t commented:
It's weird, because I updated only the system.json, not the main.yml, as you can see in the commits sections.
Well, I hope now everything is ok!
As you can see in the pull requests section, I've already updated the it.jason.

pwatson100 commented:
Superseded by new request
